pkg/controller/certificatesigningrequests/acme/acme_test.go

```diff
@@ -46,9 +51,138 @@ import (
 var (
 	fixedClockStart = time.Now()
 	fixedClock      = fakeclock.NewFakeClock(fixedClockStart)
+
+	certificatesigningrequestGVK = schema.GroupVersionKind{Group: "certificates.k8s.io", Version: "v1", Kind: "CertificateSigningRequest"}
 )
 
-func TestProcessItem(t *testing.T) {
+func Test_controllerBuilder(t *testing.T) {
+	baseCSR := gen.CertificateSigningRequest("test-csr",
+		gen.SetCertificateSigningRequestCertificate([]byte("csr")),
+	)
+
+	baseOrder := gen.Order("test-order",
+		gen.SetOrderNamespace("test-namespace"),
+	)
+
+	tests := map[string]struct {
+		existingCSR       runtime.Object
+		existingCMObjects []runtime.Object
+		givenCall         func(*testing.T, cmclient.Interface, kubernetes.Interface)
+		expectRequeueKey  string
+	}{
+		"if no request then no request should sync": {
+			existingCSR:       nil,
+			existingCMObjects: []runtime.Object{baseOrder},
+			givenCall:         func(t *testing.T, _ cmclient.Interface, _ kubernetes.Interface) {},
+			expectRequeueKey:  "",
+		},
+		"if no changes then no request should sync": {
+			existingCSR:       baseCSR,
+			existingCMObjects: []runtime.Object{baseOrder},
+			givenCall:         func(t *testing.T, _ cmclient.Interface, _ kubernetes.Interface) {},
+			expectRequeueKey:  "",
+		},
```

Comment:
suggestion: there might be a duplicate here 😅

Reply:
Thanks, I have updated the test name to make it clear it is not a duplicate.